pkg/migration/controllers/resourcetransformation.go

```diff
@@ -267,7 +272,7 @@ func (r *ResourceTransformationController) validateTransformResource(ctx context
 					Specs:            spec,
 				}
 				if err := resourcecollector.TransformResources(object, []stork_api.TransformResourceInfo{*resInfo}, metadata.GetName(), metadata.GetNamespace()); err != nil {
-					log.TransformLog(transform).Errorf("Unable to apply patch path %s on resource kind: %s/,%s/%s,  err: %v", path, kind, resInfo.Namespace, resInfo.Name, err)
+					log.TransformLog(transform).Errorf("Unable to apply patch path %s on resource kind during validation: %s/,%s/%s,  err: %v", path, kind, resInfo.Namespace, resInfo.Name, err)
```

Comment:
nit: I think "during validation" should be moved before "resource kind". We expect tp see <> resource kind: <resourceKind>

Reply:
Addressed.